packages/collabswarm/src/collabswarm-document.ts

```diff
@@ -507,4 +497,34 @@ export class CollabswarmDocument<
     // Fire change handlers.
     this._fireLocalUpdateHandlers([hash]);
   }
+
+  public async pin() {
+    // Apply local change w/ automerge.
```

Comment:
`// Apply local change w/ CRDT provider.`

Reply:
Did you mean to specify Automerge?

Reply:
Ah whoops, good catch

Reply:
```suggestion
    // Apply local change w/ CRDT provider.
```